data/lldb/monobt.py

```diff
@@ -14,15 +14,14 @@ def print_frames(thread, num_frames, current_thread):
         if function_name == "interp_exec_method_full":
             try:
                 s = 'frame->imethod->method'
-                klassname = frame.EvaluateExpression('(char*) ' + s + '->klass->name').summary[1:-1]
-                methodname = frame.EvaluateExpression('(char*) ' + s + '->name').summary[1:-1]
+                methoddesc = frame.EvaluateExpression('(char*) mono_method_full_name (' + s + ', 1)').summary[1:-1]
 
```

Comment:
Note that this will resume the runtime and execute code in it leading to all kinds of weird behaviour.

Reply:
good point, will keep it in mind. `mono_pmip` does kinda the same in the JIT case.